lib/lintPostcssResult.mjs

```diff
@@ -137,5 +138,15 @@ function isFixCompatible({ stylelint: { disabledRanges } }, name) {
  * @returns {void}
  */
 function warnDeprecatedRule(result, ruleName) {
-	result.warn(`The "${ruleName}" rule is deprecated.`, { stylelintType: 'deprecation' });
+	const message = `The "${ruleName}" rule is deprecated.`;
+
+	emitDeprecationWarning(
+		message,
+		'RULE',
+		`Please be aware that the "${ruleName}" rule will soon be either removed or renamed.`,
+	);
+
+	// @ts-expect-error
+	// https://github.com/nodejs/node/blob/main/doc/api/process.md#processnodeprecation
+	if (!process.noDeprecation) result.warn(message, { stylelintType: 'deprecation' });
```

Comment:
ref https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/process.d.ts

Reply:
It looks like a contribution chance to `@types/node`. 😃

Reply:
Be my guest.
I don't want to be the guy that will generate errors all over the word due to the `// @ts-expect-error` suddenly not being accurate anymore.
😃